org.buhocms.BuhoCMS.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.buhocms.BuhoCMS
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: buhocms
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
```

Comment:
this is an overly broad filesystem permission.

Reply:
It is used to read the user's selected folder. What would you advise to use?

Reply:
`home` is more narrow than `host` for sure